test/tests/override-cmd/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eo pipefail
+
+image="$1"
+
+# Test that we can override the CMD with echo
+hello="world-$RANDOM-$RANDOM"
+output="$(docker run --rm "$image" echo "Hello $hello")"
```

Comment:
Now that I look at this again, we may be leaving dangling volumes for some images since `docker run --rm` won't remove those. I guess we need a separate `docker rm -v`.

I don't have time to update this right now, but I can do so later today.